pkg/auth/requests/impersonate.go

```diff
@@ -29,6 +29,7 @@ func (h *impersonatingAuth) Authenticate(req *http.Request) (k8sUser.Info, bool,
 	}
 	user := userInfo.GetName()
 	groups := userInfo.GetGroups()
+	extra := userInfo.GetExtra()
```

Comment:
This can be moved down where it's needed, the other two vars are used in the below code blocks